### ISSUE TITLE
Missing time of update on top of the page

### ISSUE THREAD

**Issue by pipporobby** [poblabs/weewx-belchertown#391]
Hello,
I don't know if is a bug. Under Centos 7 and Centos 8 it doesn't show the  time of update on top.

version 1.1 and 1.2 (beta)

source code in the homepage seems ok:

// Update the Last Updated timestamp with moment.js,
            updated = moment.unix( 1599712920 ).utcOffset(-420.0).format("LL, LTS");
            updated_text = "Last Updated " + updated;
            jQuery(".updated").html( updated_text );

![HomePage](https://user-images.githubusercontent.com/67025077/92682580-891c7e80-f2e5-11ea-8746-222bf541087b.png)

**Comment by poblabs** [comment]
Link to your site?

**Comment by pipporobby** [comment]
Hello, I noticed that this happens only on my Centos installed at my pc at the office. I installed Belchertown at DigitalNation server and no problem there. Curious anyway why ! May be moment.js ?

**Comment by poblabs** [comment]
Hmm, I'm not sure why it is doing that. Very strange. Any errors when you try to debug it? https://github.com/poblabs/weewx-belchertown/tree/development#how-to-use-debug

**Comment by pipporobby** [comment]
Debug 

![Untitled](https://user-images.githubusercontent.com/67025077/92973599-60a59900-f439-11ea-8df0-28a2d466b815.png)

**Comment by pipporobby** [comment]
One more question. I would like to change the actual time zone under belchertown skin. I known moment.js offer this option (moment-timezone). How can I do it? Thanks

**Comment by poblabs** [comment]
The timezone is auto-set from your system. If you want to override it, you can change it on your system. I suppose you could also edit the belchertown.py and belchertown.js.tmpl files to override it as well. A bit more advanced and there's probably a lot to change in code. Easiest might be changing your system time and rebooting.

**Comment by pipporobby** [comment]
I have two weather stations under the same server where the timezone is America/Los_Angeles.
One weather station is in California so it's ok with the timezone. The second one is in New York (3 hours differences).
I noticed that modifing "/60"  in belchertown.py will change the "Last update" time

moment_js_utc_offset = (calendar.timegm(moment_js_stop_struct) - calendar.timegm(time.gmtime(time.mktime(moment_js_stop_struct))))/60

My question is if this is a correct way to change the Last update time. I tried with different numbers e.g. /240 /480 I don't understand the rules fully. Is this a moment.js option?

Thanks

p.s I know about DST problem
